### PR TITLE
Remove old and add new ICQ/AOL plugin

### PR DESCRIPTION
**Pull request by Herrie82**
Old ICQ/AOL plugin is no longer working, therefore remove them. 

Add new icyque plugin that supports the new WIM protocol.

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>

### CODE REVIEW

File: meta-luneos/recipes-support/pidgin/icyque_git.bb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+SUMMARY = "WIM Protocol plugin for ICQ for Adium, Pidgin, Miranda and Telepathy IM Framework"
+SECTION = "webos/services"
+LICENSE = "GPLv3.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=1ebbd3e34237af26da5dc08a4e440464"
+
+DEPENDS = "pidgin json-glib"
+
+inherit pkgconfig
+
+SRC_URI = "git://github.com/EionRobb/icyque;branch=master;protocol=git"
```
Comment (shr-project) [inline meta-luneos/recipes-support/pidgin/icyque_git.bb]: set PV

Comment (shr-project) [reply meta-luneos/recipes-support/pidgin/icyque_git.bb]: And please drop the default ";branch=master;protocol=git"

Comment (Herrie82) [reply meta-luneos/recipes-support/pidgin/icyque_git.bb]: Done & done. Was copied from another recipe. Will clean those up too in another PR.

File: meta-luneos/recipes-support/pidgin/icyque_git.bb [hunk 2]
```diff
@@ -0,0 +1,27 @@
+SUMMARY = "WIM Protocol plugin for ICQ for Adium, Pidgin, Miranda and Telepathy IM Framework"
+SECTION = "webos/services"
+LICENSE = "GPLv3.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=1ebbd3e34237af26da5dc08a4e440464"
+
+DEPENDS = "pidgin json-glib"
+
+PV = "0.1.0+gitr${SRCPV}"
```
Comment (shr-project) [inline meta-luneos/recipes-support/pidgin/icyque_git.bb]: According to https://github.com/EionRobb/icyque/blob/master/libicyque.c#L1734 it should be "0.1+git${SRCPV}"

Comment (Herrie82) [reply meta-luneos/recipes-support/pidgin/icyque_git.bb]: I really just put a random number because I didn't find any releases/tags. But updated now ;)

**Comment by shr-project** [comment]
LGTM

**Pull request merged**